clang/lib/Format/Format.cpp

```diff
@@ -946,6 +952,51 @@ template <> struct MappingTraits<FormatStyle> {
     bool SpacesInCStyleCastParentheses = false;
     bool SpacesInParentheses = false;
 
+    if (IO.outputting()) {
+      IO.mapOptional("AlignAfterOpenBracket", Style.AlignAfterOpenBracket);
+    } else {
+      // For backward compatibility.
+      BracketAlignmentStyle local = BAS_Align;
```

Comment:
```suggestion
      BracketAlignmentStyle Local = BAS_Align;
```
Although there might be a better name than local, but currently I don't have one.